File: Java/Basics To Java/src/F1/F1_10.java
package F1;

import java.util.Scanner;

public class F1_10 {
	public static void mainx(String[] args) {
		Scanner sc = new Scanner(System.in);

		String vystup;
		String text;
		System.out.println("zadej text s mezerama");
		text = sc.nextLine();
		vystup = bezmezerF(text); // funkce
		System.out.println(vystup); // funkce
	}
	public static String bezmezerF(String veta) // funkce
	{
		veta = veta.replace(" ", "");
		return veta;
	}
}

File: C#/Basic logic/F1/F1_10.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace F1
{
    internal class F1_10
    {
        static void Mainx(string[] args)
        {
            string vystup;
            string text, sub;
            Console.WriteLine("zadej text s mezerama");
            text = Console.ReadLine();
            vystup = bezmezerF(text); //funkce
            bezmezerS(text, out sub); //subrutina
            Console.WriteLine(vystup); //funkce
            Console.WriteLine(sub); //subrutina
        }

        static string bezmezerF(string veta) //funkce
        {
            //string presun = "";
            //char pismeno;

            //for (int i = 0; i < veta.Length ; i++)
            //{
            //    if (veta[i] == ' ')
            //        pismeno = 'x';
            //    else
            //        pismeno = veta[i];
            //    presun = presun + pismeno;
            //}
            veta = veta.Replace(" ", string.Empty);
            return veta;
        }

        static void bezmezerS(string veta, out string s) //subrutina
        {
            veta = veta.Replace(" ", string.Empty);
            s = veta;
        }
    }
}
